Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -121,11 +121,13 @@ private function getFunctionyTokens()
     }
 
     /**
-     * Gets the name of tokens that are actually language construction.
+     * Gets the token kinds of actually language construction.
      *
+     * @var int[] $languageConstructionTokens Token kinds.
```

Comment:
> do you agree withj fabbot here?

Yes and no. It's changes are wrong, but only because your code is not allowed there anyway.